README.md

```diff
@@ -50,3 +50,17 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+The time complexity for this would be $$O((n + n + n + n) * n)$$, which simplifies to $$O(n^2)$$. This is because there are loops within every single function, but then each of these functions are called within another loop. You could see it as $$O(4n * n)$$, but of course we can just ignore the 4. The memory complexity would be O(n) just because it uses simple temporary arrays, and that's about it. Within the main loop, it calls other functions to perform calculations, but the main loop itself, makes decisions based on the distance between nodes, and tries to find the most optimal path. This is where is has to loop through to check several versions of paths against others.
```

Comment:
Your complexity is wrong -- how many times is the loop running and how much work are you doing inside the loop?